test/parallel/test-http-allow-req-after-204-res.js

```diff
@@ -39,17 +39,13 @@ const server = http.createServer(common.mustCall((req, res) => {
 }, codes.length));
 
 function nextRequest() {
-  const method = methods.shift();
 
-  const request = http.request({
+  const request = http.get({
     port: server.address().port,
-    method: method,
     path: '/'
   }, common.mustCall((response) => {
     response.on('end', common.mustCall(() => {
-      if (methods.length === 0) {
-        server.close();
-      } else {
+      if (countdown.dec()) {
```

Comment:
No strong opinion, but instead of modifying `common/countdown.js`, you can just use the existing API and do something like

```js
countdown.dec();
if (countdown.remaining > 0) {
  ...
}
```

Reply:
@aqrln : I feel making the change in `countdown.js`will be better as it makes the test case condition more readable and save an extra line! Kindly suggest. Also, could you please run the CI. I've fixed the lint issues. Thanks.

Reply:
@mithunsasidharan sure, no problem. In the future, please run `make -j4 test` locally before pushing to the PR branch to save your time and the number of review iterations. Thanks!